plugins/ModuleLevelTrigger.cpp

```diff
@@ -102,7 +103,9 @@ ModuleLevelTrigger::do_start(const nlohmann::json& startobj)
   m_paused.store(true);
   m_running_flag.store(true);
 
-  m_token_manager.reset(new TokenManager(m_token_source, m_initial_tokens, m_run_number));
+  m_livetime_counter.reset(new LivetimeCounter(LivetimeCounter::State::kPaused));
```

Comment:
I believe this is inside the `start` command handler so I expect the state to be set to `kLive`.

But, either way, I see one must implicitly assume some overall DAQ state machine in order to know what `k*` to set.  

Modules are supposed to be agnostic to the overall FSM, right?

Reply:
I'm still not really sure how the overall FSM is supposed to relate (or not) to individual modules. The MLT is definitely the place that knows about whether triggering is live/dead, so I think it's the right place for counting those times.

I copied the behaviour where we're paused after the "start" command from `TriggerDecisionEmulator` (the "resume" command then enables triggers). I don't quite recall the rationale for having it this way, but no one has complained about it, so I'm inclined to keep it this way for the moment.